qcodes/instrument_drivers/AlazarTech/ATS.py

```diff
@@ -639,6 +647,34 @@ def acquire(self, mode=None, samples_per_record=None,
         Returns:
             Whatever is given by acquisition_controller.post_acquire method
         """
+        loop : asyncio.AbstractEventLoop = asyncio.get_event_loop()
+        return loop.run_until_complete(self.async_acquire(
+            mode=mode,
+            samples_per_record=samples_per_record,
+            records_per_buffer=records_per_buffer,
+            buffers_per_acquisition=buffers_per_acquisition,
+            channel_selection=channel_selection,
+            transfer_offset=transfer_offset,
+            external_startcapture=external_startcapture,
+            enable_record_headers=enable_record_headers,
+            alloc_buffers=alloc_buffers,
+            fifo_only_streaming=fifo_only_streaming,
+            interleave_samples=interleave_samples,
+            get_processed_data=get_processed_data,
+            allocated_buffers=alloc_buffers,
+            buffer_timeout=buffer_timeout,
+            acquisition_controller=acquisition_controller
+        ))
+
+    async def async_acquire(self, mode=None, samples_per_record=None,
+                records_per_buffer=None, buffers_per_acquisition=None,
+                channel_selection=None, transfer_offset=None,
+                external_startcapture=None, enable_record_headers=None,
+                alloc_buffers=None, fifo_only_streaming=None,
+                interleave_samples=None, get_processed_data=None,
+                allocated_buffers=None, buffer_timeout=None,
+                acquisition_controller=None) -> np.ndarray:
```

Comment:
The acquire method can return anything. acquisition controller's post_acquire defines this.
```suggestion
                acquisition_controller=None):
```